src/main/static/src/components/common/select/OrganisaatioSelect.tsx

```diff
@@ -92,10 +101,14 @@ export class OrganisaatioSelect extends React.Component<OrganisaatioSelectProps,
             : null;
     };
 
+    _resolveName = (oid: string, locale: string): string => this.props.organisationNames?.[oid]?.[locale] || oid;
+
     _renderParents = (organisaatio: OrganisaatioSelectObject): React.ReactNode => {
-        return organisaatio.parentNames.map((parent: string, index: number) => (
-            <span key={index} className="parent">
-                {parent} &gt;{' '}
+        const path = organisaatio.oidPath.split('/');
```

Comment:
Why has this changed? I think a better place might be somewhere in https://github.com/Opetushallitus/henkilo-ui/blob/08b7440b67ebc6fcb262ee6d3cbb4081cc2bd138/src/main/static/src/utilities/organisaatio.util.tsx#L53

Reply:
to keep impact minimal. currently the only known usage is within this component. given the non-existent test coverage, didn't want to touch functions manipulating common datastructures